lib/stack_master/parameter_resolvers/one_password.rb

```
@@ -0,0 +1,49 @@
+module StackMaster
+  module ParameterResolvers
+    class OnePassword < Resolver
+      OnePasswordNotFound = Class.new(StandardError)
+
+      array_resolver
+
+      def initialize(config, stack_definition)
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def resolve(params={})
+        raise RuntimeError, "1password requires the `OP_SESSION_<name>` to be set" if ENV.keys.grep(/OP_SESSION_\w+$/).empty?    
+        get_items(params)
+      end
+
+      private
+
+      def op_get_item(item, vault)
+        begin
+          item = %x(op get item --vault='#{vault}' '#{item}' 2>&1)
+          return item if JSON.parse(item)
```

I think this would swallow any potential JSON parsing errors...